lib/contestant.rb

```diff
@@ -0,0 +1,26 @@
+class Contestant
+
+  attr_reader :first_name, :last_name, :age, :state_of_residence,
+              :spending_money, :game_interests
+
+  def initialize(init_hash)
```

Comment:
Horrid naming here. The parameter should be named `contestant_info` or `attributes`.